utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
+% Copyright 2023 The MathWorks, Inc.
+prjFileName="fsdaToolboxPackaging.prj";
 toolboxOption=matlab.addons.toolbox.ToolboxOptions(prjFileName);
 
 % Update the version number
```

Comment:
Line 17: there should be a spacing after "%" before start of the comment text

% Package the FSDA toolbox

Reply:
Added space after %